Formula/halide.rb

```diff
@@ -4,6 +4,7 @@ class Halide < Formula
   url "https://github.com/halide/Halide/archive/v13.0.0.tar.gz"
   sha256 "5ab7d5d9bc04ab0902a88751e535ff231ab9da1911e46f272a52cf41131609f3"
   license "MIT"
+  revision 1
```

Comment:
https://github.com/halide/Halide/releases
```suggestion
  url "https://github.com/halide/Halide/archive/v13.0.1.tar.gz"
  sha256 "TBD"
  license "MIT"
```

Reply:
That's for a separate PR. This one will be waiting a while for the self-hosted Linux runner.

Reply:
Please rebase to pick up #89931